frontend/learning/components/CoreLearning/CoreLearning.js

```diff
@@ -74,9 +72,18 @@ export default function CoreLearning(props) {
         </>
     );
 
+    const showAssignmentPage = () => {
+        props.history.push(`/learning/assignment/${qid}`);
+    };
+
     const renderReviewQuestion = () => (
         <>
-            <ProgressBar {...progressBar} />
+            <div className={'progressBarContainer'}>
+                <button className={'exitButton'} href={`/learning/assignment/${qid}`}> {'\u{2717}'} </button>
```

Comment:
`href` doesn't work for button, use `onClick={() => {window.location = '/learning/xxx';};}`

Reply:
I changed it to an onclick function and it works currently. But it requires reloading the page. I suggest we should consider adding a router for the long run.